datalad_container/containers_list.py

```diff
@@ -44,9 +44,11 @@ def __call__(dataset=None):
                              purpose='list containers')
 
         for sub in ds.subdatasets(return_type='generator'):
-            for c in Dataset(sub['path']).containers_list():
-                c['name'] = sub['gitmodule_name'] + '/' + c['name']
-                yield c
+            subds = Dataset(sub['path'])
+            if subds.is_installed():
```

Comment:
I wonder if it is worth collecting the list of those sub-datasets which aren't installed and issuing `lgr.info` at the end with the message like `"%d subdatasets which might contain containers we found not installed."`

Reply:
True. Will do. Once at it, I should also check `run-procedure --discover` for this issue and treat it similarly.